src/components/application_manager/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -266,49 +268,71 @@ void ResetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
     }
   }
 
-  if (!IsPendingResponseExist()) {
-    bool result =
-        ((hmi_apis::Common_Result::SUCCESS == ui_result_) &&
-         (hmi_apis::Common_Result::SUCCESS == tts_result_ ||
-          hmi_apis::Common_Result::UNSUPPORTED_RESOURCE == tts_result_)) ||
-        ((hmi_apis::Common_Result::SUCCESS == ui_result_) &&
-         (hmi_apis::Common_Result::INVALID_ENUM == tts_result_)) ||
-        ((hmi_apis::Common_Result::INVALID_ENUM == ui_result_) &&
-         (hmi_apis::Common_Result::SUCCESS == tts_result_));
-
-    mobile_apis::Result::eType result_code;
-    const char* return_info = NULL;
-
-    if (result) {
-      if (hmi_apis::Common_Result::UNSUPPORTED_RESOURCE == tts_result_) {
-        result_code = mobile_apis::Result::WARNINGS;
-        return_info =
-            std::string("Unsupported phoneme type sent in a prompt").c_str();
-      } else {
-        result_code = static_cast<mobile_apis::Result::eType>(
-            std::max(ui_result_, tts_result_));
-      }
-    } else {
-      result_code = static_cast<mobile_apis::Result::eType>(
-          std::max(ui_result_, tts_result_));
-    }
+  if (IsPendingResponseExist()) {
```

Comment:
@dcherniev, Are you that is correct? As I read function name, if  `IsPendingResponseExist` return false then print error and etc.